tests/exec-steps.sh

```diff
@@ -60,19 +60,21 @@ for STEP in /var/lib/tests/step-*.sh; do
         echo "! Uninstall failed in $STEP, output:"
         echo "$UNINSTALL_OUTPUT"
         FAILED=$((FAILED + 1))
+        break # Fail es early as possible
```

Comment:
I think it's actually better to fail at the end so we can see all the failures together rather than only the first one.
Having said that, I'm OK to undo this in the next change.

Reply:
Ok, the problem with continuation is, that it might introduce false positives and vice versa. 
But I am fine with removing the `break`.